From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 29 Jan 2015 10:46:40 +0100
Subject: mime_util: force text/x-suse-ymp to be downloaded

YMP files (YaST One Click Install) are plaintext XML, but also not
very interesting in themselves. Force them to be stored.

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 third_party/blink/common/mime_util/mime_util.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/blink/common/mime_util/mime_util.cc b/third_party/blink/common/mime_util/mime_util.cc
index 8e4c00a44fbe250e5fb002aea97064688943adcc..b07c2e667b9c974fb3d15f87bcdd9d6853e6410d 100644
--- a/third_party/blink/common/mime_util/mime_util.cc
+++ b/third_party/blink/common/mime_util/mime_util.cc
@@ -88,6 +88,7 @@ constexpr auto kUnsupportedTextTypes =
         "text/x-csv",
         "text/x-vcf",
         "text/rtf",
+        "text/x-suse-ymp",
         "text/comma-separated-values",
         "text/csv",
         "text/tab-separated-values",
